cognite/client/_api/simulators/models_revisions.py

```diff
@@ -36,14 +36,16 @@ def list(
         self,
         limit: int = DEFAULT_LIMIT_READ,
         sort: PropertySort | None = None,
-        filter: SimulatorModelRevisionsFilter | dict[str, Any] | None = None,
+        model_external_ids: str | Sequence[str] | None = None,
+        all_versions: bool | None = None,
```

Comment:
Here we could add as well the filters on createdTime and lastUpdatedTime as it's supported in the API: https://api-docs.cognite.com/20230101/tag/Simulator-Models/operation/filter_simulator_model_revisions_simulators_models_revisions_list_post
```suggestion
        all_versions: bool | None = None,
        created_time: TimestampRange | None = None,
        last_updated_time: TimestampRange | None = None,
```